aspnetcore/security/authentication/add-user-data.md

```diff
@@ -22,15 +22,37 @@ The project sample is created from a Razor Pages web app, but the instructions a
 
 ## Prerequisites
 
+::: moniker range=">= aspnetcore-3.0"
+
+[!INCLUDE [](~/includes/3.0-SDK.md)]
+
+::: moniker-end
+
+::: moniker range="< aspnetcore-3.0"
+
 [!INCLUDE [](~/includes/2.2-SDK.md)]
 
+::: moniker-end
+
 ## Create a Razor web app
 
 # [Visual Studio](#tab/visual-studio)
 
 * From the Visual Studio **File** menu, select **New** > **Project**. Name the project **WebApp1** if you want to it match the namespace of the [download sample](https://github.com/aspnet/AspNetCore.Docs/tree/live/aspnetcore/security/authentication/add-user-data) code.
 * Select **ASP.NET Core Web Application** > **OK**
+
+::: moniker range=">= aspnetcore-3.0"
+
+* Select **ASP.NET Core 3.0** in the dropdown
```

Comment:
Put the entire bullet list inside the moniker range blocks. They don't work with individual list items.